sabnzbd/filesystem.py

```diff
@@ -472,12 +472,17 @@ def fix_unix_encoding(folder: str):
                         logging.info("Cannot correct name of %s", os.path.join(root, name))
 
 
+def is_valid_script(basename: str) -> bool:
+    """ Determine if 'basename' is a valid script """
+    return basename in sabnzbd.api.list_scripts(default=False, none=False)
```

Comment:
Could you maybe move `list_scripts` to `filesystem.py` since it's now not an api-specific function anymore?